core/src/test/scala/unit/kafka/log/LogTest.scala

```diff
@@ -505,7 +575,8 @@ class LogTest extends JUnitSuite {
   @Test
   def testMessageSizeCheck() {
     val first = MemoryRecords.withRecords(CompressionType.NONE, Record.create("You".getBytes), Record.create("bethe".getBytes))
-    val second = MemoryRecords.withRecords(CompressionType.NONE, Record.create("change (I need more bytes)".getBytes))
+    val second = MemoryRecords.withRecords(CompressionType.NONE,
```

Comment:
I had to increase the size of the second message significantly to get this test to pass. Not sure why the baseline message size should be so much larger in this rebase.